react/features/prejoin/actions.js

```diff
@@ -232,27 +186,15 @@ export function dialOut(onSuccess: Function, onFail: Function) {
  */
 export function initPrejoin(tracks: Object[], errors: Object) {
     return async function(dispatch: Function) {
-        const audioTrack = tracks.find(t => t.isAudioTrack());
-        const videoTrack = tracks.find(t => t.isVideoTrack());
-
         dispatch(setPrejoinDeviceErrors(errors));
 
-        if (audioTrack) {
-            dispatch(addPrejoinAudioTrack(audioTrack));
-        } else {
-            dispatch(setAudioDisabled());
-        }
+        const hasVideoTrack = Boolean(tracks.find(t => t.isVideoTrack()));
```

Comment:
While we are here: this check is kinda wrong. It means you cannot re-enable video if you have the `startWithAudioMuted` option set in config.js or the URL. Since the user is in full control, they should be able to enable it.

The one time where it shouldn't is if startSilent was set.

Reply:
Solved this.